src/app/individualProduct.php

```diff
@@ -0,0 +1,84 @@
+<?php
```

Comment:
Issue with sidebar on mobile - reduce font size

Reply:
please

Reply:
This is done, but had to modify class, please check navbar across 3 pages to confirm this has not broken anything.